spec/middleware/authentication_spec.rb

```diff
@@ -16,6 +16,15 @@
 
         expect(status).to eq(200)
       end
+
+      it 'allows the request if given a correct password header and a Passwords header' do
+        app = lambda { |env| [200, {'Content-Type' => 'application/json'}, Array.new] }
```

Comment:
Unused block argument - `env`. If it's necessary, use `_` or `_env` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.<br>Line is too long. [87/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.